src/middlwares/authentication.middleware.ts

```diff
@@ -0,0 +1,37 @@
+import * as jwt from 'jsonwebtoken';

```

Comment:
@AimenBaseer please use [Guards](https://docs.nestjs.com/guards) as we are using NestJS and this is what they provide for authentication. Guards can use different strategies please refer [here](https://docs.nestjs.com/security/authentication)